code/modules/projectiles/guns/projectile/pistol.dm

```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/gun/projectile/automatic/pistol
 	name = "syndicate pistol"
-	desc = "A small, easily concealable 10mm handgun. Has a threaded barrel for suppressors."
+	desc = "Known as the DC3 Elite, this is a small, easily concealable 10mm handgun. Has a threaded barrel for suppressors."

```

Comment:
Not really but it affects absolutely nothing. If you're wondering, it's a reference to a gun in the game Black, which had a gun which was a hybrid of a Beretta M9 and Desert Eagle, which was named the DC3 Elite. Just thought this'd be a neat little reference + it sounds cool.